yoti_python_sdk/profile.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+import collections
+import json
+
+from yoti_python_sdk import config
+from yoti_python_sdk.anchor import Anchor
+from yoti_python_sdk.attribute import Attribute
+from yoti_python_sdk.protobuf.v1.protobuf import Protobuf
+
+
+class Profile:
+    def __init__(self, profile_attributes):
+        self.profile = {}
+
+        if profile_attributes:
+            for field in profile_attributes:
+                value = Protobuf().value_based_on_content_type(
+                    field.value,
+                    field.content_type
+                )
+
+                anchors = Anchor().parse_anchors(field.anchors)
+
+                self.profile[field.name] = Attribute(field.name, value, anchors)
+                # self.profile[field.name] = Attribute(field.name, field.value, anchors)
+
+                if field.name.startswith(config.ATTRIBUTE_AGE_OVER) or field.name.startswith(

```

Comment:
I would rather use a regex here to capture `age_(Over|Under)` as there could be more derivation on age in the future.

Reply:
As discussed, we'll leave this as is, as any future derivations on age are likely to be in a separate attribute, so both of these approaches are fine.